sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/SpecializedBlobClientBuilder.java

```diff
@@ -86,7 +90,8 @@ public BlockBlobClient buildBlockBlobClient() {
 
     /**
      * Creates a {@link BlockBlobAsyncClient} based on options set in the Builder. BlockBlobAsyncClients are used to
-     * perform generic upload operations such as {@link BlockBlobAsyncClient#uploadFromFile(String) upload from file}
+     * perform generic upload operations such as {@link BlockBlobAsyncClient#uploadFromFile(String, Integer,
```

Comment:
This method was removed from the block blob client, instead reference it's upload method,